### ISSUE TITLE
CLIENTVERSION error for Sweden

### ISSUE THREAD

**Issue by regagain** [schildbach/public-transport-enabler#266]
Using Transportr 2.0.6.

![Screenshot_20190619-090647_Transportr](https://user-images.githubusercontent.com/1060371/59760711-39dad000-9293-11e9-94cf-25a416648524.png)

**Comment by schildbach** [comment]
Please don't post screenshots! Rather, can you post the full stacktrace in textual form? Thanks.

**Comment by regagain** [comment]
Sure:
```java
java.lang.RuntimeException: CLIENTVERSION 
de.schildbach.pte.AbstractHafasClientInterfaceProvider.jsonTripSearch(AbstractHafasClientInterfaceProvider.java:579)
de.schildbach.pte.AbstractHafasClientInterfaceProvider.queryTrips(AbstractHafasClientInterfaceProvider.java:209)
de.schildbach.pte.AbstractNetworkProvider.queryTrips(AbstractNetworkProvider.java:92)
```

**Comment by schildbach** [comment]
This was fixed via 2182639115fe2180f7fcc2230402abece66956f3.
